hieroplatform/src/test/java/org/hiero/sketch/MembershipTest.java

```diff
@@ -22,12 +22,16 @@ public void TestFullMembership() {
         final IRowIterator IT = FM.getIterator();
         int tmp = IT.getNextRow();
         while (tmp >= 0) {
-            //System.out.print(tmp+",");
             tmp = IT.getNextRow();
         }
         assertEquals(this.size, FM.getSize());
         assertTrue(FM.isMember(7));
         assertFalse(FM.isMember(20));
+        final FullMembership FM1 = new FullMembership(1000);
+        final IMembershipSet FM2 = FM1.sample(0.1);
+        assertEquals(100, FM2.getSize());
```

Comment:
This is also surprising. Maybe we want two sampling methods: sampleExact and sampleApproximate?
I don't mind just sampleExact if it does not incur an extra significant cost.